### ISSUE TITLE
Language-agnostic BERT Sentence Embedding

### ISSUE THREAD

**Issue by onikazu** [onikazu/ArticleSum#40]
## 0. 論文
https://arxiv.org/pdf/2007.01852v1.pdf

## 1. どんなもの？
109言語に対応した文レベルの埋め込み表現を出力するモデルの提案。
MLM + TLM + dual encoderのあわせ技とのこと

更に深堀りして関連研究を見ていく必要あり

## 6. 次に読むべき論文は？
sentence-bert
https://arxiv.org/abs/1908.10084

LANGUAGE-AGNOSTIC MULTILINGUAL MODELING
https://arxiv.org/pdf/2004.09571.pdf

Improving Multilingual Sentence Embedding using Bi-directional Dual Encoder with Additive Margin Softmax(Dual encoder)
https://arxiv.org/abs/1902.08564

Cross-lingual Language Model Pretraining(TLM)
http://papers.nips.cc/paper/8928-cross-lingual-language-model-pretraining.pdf
